static/js/src/cve/cve.js

```diff
@@ -148,3 +152,12 @@ tooltipIconList.forEach(function (tooltipIcon) {
     false
   );
 });
+
+function toggleCVEList() {
```

Comment:
To me the word toggle means switching from one state to another and back. In this case though there is only one switch in case there is a parameter `status` present. Should we still be calling this function `toggle` or will the actual switching back come later?

Reply:
I added this mostly for testing as the cve cards only exist for recent cves in this current branch (ie, I still needed to use the table to show filtered releases), but yes I think you're right about toggle maybe not being the best verb here. I'll update!